consumers/src/main/java/uk/gov/moj/sdt/cmc/consumers/api/IJudgement.java

```diff
@@ -0,0 +1,12 @@
+package uk.gov.moj.sdt.cmc.consumers.api;
+
+import uk.gov.moj.sdt.cmc.consumers.request.judgement.JudgementRequest;
+import uk.gov.moj.sdt.cmc.consumers.response.judgement.JudgementResponse;
+
+public interface IJudgement {
```

Comment:
To avoid confusion with the request type and make it more specific, rename this interface to ```IJudgmentService```

Reply:
done